src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderProfileDetail.java

```diff
@@ -70,7 +70,7 @@ public static class PK implements Serializable {
         private Integer sequence;
     }
 
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
Change for hibernate upgrade, needed as sometimes there is no profile code (e.g. free text for `NATIO` type).